Description: No linux-gate.so dependency
 If we rebuild an rpm, don't generate a dependancy on linux-gate.so as
 that is a virtual shared library generated by 2.6 kernels so that can
 decide upon the best mechanism to transition from user to kernelspace
Author: Ken Schalk <ken@vanguard.xorian.net>
diff --git a/autodeps/linux.req b/autodeps/linux.req
index 6efa990..506788d 100644
--- a/autodeps/linux.req
+++ b/autodeps/linux.req
@@ -47,7 +47,7 @@ if [ "$needed" -eq 0 ]; then
     lib64=`if file -L $f 2>/dev/null | \
 	grep "ELF 64-bit" >/dev/null; then echo "$mark64"; fi`
     ldd $f | awk '/=>/ {
-	if ($1 !~ /libNoVersion.so/ && $1 !~ /4[um]lib.so/ && $1 !~ /libredhat-kernel.so/) {
+	if ($1 !~ /libNoVersion.so/ && $1 !~ /4[um]lib.so/ && $1 !~ /libredhat-kernel.so/ && $1 !~ /linux-gate.so/) {
 	    gsub(/'\''"/,"\\&",$1);
 	    printf "%s'$lib64'\n", $1
 	}
@@ -61,7 +61,7 @@ if [ "$needed" -eq 0 ]; then
     lib64=`if file -L $f 2>/dev/null | \
 	grep "ELF 64-bit" >/dev/null; then echo "$mark64"; fi`
     ldd $f | awk '/=>/ {
-	if ($1 !~ /libNoVersion.so/ && $1 !~ /4[um]lib.so/ && $1 !~ /libredhat-kernel.so/) {
+	if ($1 !~ /libNoVersion.so/ && $1 !~ /4[um]lib.so/ && $1 !~ /libredhat-kernel.so/ && $1 !~ /linux-gate.so/) {
 	    gsub(/'\''"/,"\\&",$1);
 	    printf "%s'$lib64'\n", $1
 	}
